pwndbg/wrappers/file.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import pwndbg.wrappers
+
+cmd_name = "file"
+
+@pwndbg.wrappers.OnlyWithCommand(cmd_name)
+def is_statically_linked():
+
+    local_path = pwndbg.file.get_file(pwndbg.proc.exe)
+    cmd = [is_statically_linked.cmd_path, local_path]
+
+    file_out = pwndbg.wrappers.call_cmd(cmd)
+
+    if "statically" in file_out:
```

Comment:
Yeah, we can remove the check for being statically compiled. Probably we could check for it differently - looking for got/plt sections using readelf. However, the solution proposed by you is fine imho (if someone lacks the check he/she can add it :P).


Jfyi about checking for statically compiled binary - https://stackoverflow.com/questions/8040631/checking-if-a-binary-compiled-with-static says it can be done with `ldd` but I haven't checked it.